packages/babel-highlight/src/index.ts

```diff
@@ -1,10 +1,15 @@
+/// <reference path="../../../lib/third-party-libs.d.ts" />
```

Comment:
I think, this might be an issue when publishing generated `d.ts` files later - this path would be incorrect relative to generated file

Reply:
This is just while we have dual packages for Babel 7 and Babel 8 - it will be removed before starting to publish d.ts files.

Reply:
Oh well, we could put the third party `.d.ts` file in this package just to be sure that everything works if we and up publishing ts definitions before removing dual dependencies.